xgi/generators/classic.py

```diff
@@ -154,3 +157,59 @@ def trivial_hypergraph(n=1, create_using=None, default=None):
     H.add_nodes_from(nodes)
 
     return H
+
+
+def complete_hypergraph(N, order=None, max_order=None, include_singletons=False):
+    """
+    Generate a complete hypergraph, i.e. one that contains all possible hyperdges
+    at a given `order` or up to a `max_order`.
+
+    Parameters
+    ----------
+
+    N : int
+        Number of nodes
+    order : int or None
+        If not None (default), specifies the single order for which to generate hyperedges
+    max_order : int or None
+        If not None (default), specifies the maximum order for which to generate hyperedges
+    include_singletons : bool
+        Whether to include singleton edges (default: False). This argument is discarded
+        if max_order is None.
+
+    Return
+    ------
+    Hypergraph object
+        A complete hypergraph with `N` nodes
+
+    Note
+    ----
+    Only one of `order` and `max_order` can be specified by and int (not None).
+    Additionally, at least one of either must be specified.
+    """
+    # this import needs to happen when the function runs, not when the module is first
+    # imported, to avoid circular imports
+    import xgi
+
+    if bool(order) == bool(max_order):
```

Comment:
The current version does not catch if both are specified as zero.
```suggestion
    if order is not None and max_oeder is not None:
```

Reply:
Hahah I knew you were gonna make that comment. 
What I am doing trying to do is catching both those cases: (None, None), and (int, int).
Both cases are invalid and should raise the error.

I noticed it wouldn't catch 0 but thought using 0 for order or max_order is not useful anyway?

I can do a more verbose version with your line and another so that everything is explicit if you think that's better.

Reply:
I see. Yeah in general I think it's better to be explicit rather than implicit. Zen of python and what not,